components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -73,7 +73,7 @@ object ErrorPages {
     ): String {
         val title = context.getString(errorType.titleRes)
         val button = context.getString(errorType.refreshButtonRes)
-        val description = context.getString(errorType.messageRes)
+        val description = context.getString(errorType.messageRes, uri)
```

Comment:
Do we know for sure that all description messages must include the `uri`? cc @jonalmeida

Reply:
This should be safe. We do the same thing in the [function above](https://github.com/mozilla-mobile/android-components/pull/6385/files#diff-00750c22f22d2046f9c02e30915d0f31R43).

Reply:
This is in the doc for java.util.Formatter : 

>  * <p> If there are more arguments than format specifiers, the extra arguments are ignored.